internal/ethapi/api.go

```diff
@@ -1889,6 +1889,36 @@ func (api *PrivateDebugAPI) SetHead(number hexutil.Uint64) {
 	api.b.SetHead(uint64(number))
 }
 
+// BurnedETH returns the amount of burned ether for a specific
+// range of blocks under eip-1559.
+// The returned value is in denoted in Wei.
+func (api *PrivateDebugAPI) BurnedETH(start, end *hexutil.Uint64) (*hexutil.Big, error) {
```

Comment:
Nitpick: the returned sum is denominated in `wei`, so caling it `BurnedETH` is wrong, should be `BurnedWEI`

Reply:
Just call it `burned`? It's gonna be `eth_burned` anyway. Wei seems correct to me